boards/BMS/Src/Io/Io_LTC6813/Io_NewLTC6813.h

```diff
@@ -0,0 +1,104 @@
+#pragma once
+
+#include <stdbool.h>
+#include <stm32f3xx_hal.h>
+
+// Conversion factor used to convert raw voltages (100µV) to voltages (V)
+#define V_PER_100UV (1E-4f)
+
+enum CmdFormat
+{
+    CMD_WORD = 0U,
+    CMD_PEC15,
+    NUM_OF_CMD_WORDS,
+};
+#define TOTAL_NUM_CMD_BYTES (NUM_OF_CMD_WORDS << 1U)
+
+enum RegGroupByteFormat
+{
+    REG_GROUP_BYTE_0 = 0U,
+    REG_GROUP_BYTE_1,
+    REG_GROUP_BYTE_2,
+    REG_GROUP_BYTE_3,
+    REG_GROUP_BYTE_4,
+    REG_GROUP_BYTE_5,
+    REG_GROUP_PEC0,
+    REG_GROUP_PEC1,
+    NUM_REG_GROUP_BYTES,
+};
+#define NUM_OF_BYTES_IN_REG_GROUP (6U)
```

Comment:
`NUM_REG_GROUP_BYTES` and `NUM_OF_BYTES_IN_REG_GROUP` sound the same but have different values. Can we name them differently?

Reply:
yeah thats a good point